spec/card_spec.rb

```diff
@@ -2,25 +2,19 @@
 
 describe Card do
   before do
-    @card = Card.create(original_text: "How are you?", translated_text: "Kamusta?", review_date: DateTime.new(2017, 2, 24))
+    @card = Card.create(original_text: "How are you?", translated_text: "Kamusta?", review_date: DateTime.now)
   end
 
   describe "card class methods" do
     describe "self.due_today" do
       it "should know that 1 card is due today" do
-        Card.due_today.count.should == 1
+        expect(Card.due_today.count).to eq(1)
       end
     end
 
     describe "self.random" do
       it "should return the exact same card" do
-        Card.random.object_id == @card.object_id
-      end
-    end
-
-    describe "self.random" do
-      it "returns nil if it's empty" do
-        Card.none.random.should.nil?
+        expect(Card.random.id).to eq(@card.id)
```

Comment:
@kennyfrc useless test, actually =) 
You could remove it completely, because random is random :) and if you will have 2+ objects – it will not work, because random :)